test/new-e2e/go.mod

```diff
@@ -234,7 +234,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.28.0 // indirect
 	go.starlark.net v0.0.0-20230525235612-a134d8f9ddca // indirect
 	go.uber.org/atomic v1.11.0 // indirect
-	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56 // indirect
+	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
```

Comment:
Can you run `go mod tidy` in this directory? Normally the `indirect` requires are in a separate block.

Reply:
Ran it twice, it leaves it there for some reason.